tests/user.test.js

```diff
@@ -6,18 +6,20 @@ const { user0Id, user0, setupDatabase } = require('./fixtures/db')
 beforeEach(setupDatabase)
 
 test('Should signup a new user', async() => {
```

Comment:
same issues as here
https://github.com/sychova/task-manager-api/pull/1

Reply:
Could you please clarify which issue exactly do you mean?

Reply:
Fixed.